templates/field/field--node--created.html.twig

```diff
@@ -0,0 +1,3 @@
+{% include "@atoms/date-time/date-time.twig" with {
+  date_time__start: element['#object'].getCreatedTime|date('Y-m-d'),
```

Comment:
I had to look this up as I didn't know what `element['#object']` would do. Glad this exists.